src/main/java/com/unstoppabledomains/config/network/model/Location.java

```diff
@@ -0,0 +1,41 @@
+package com.unstoppabledomains.config.network.model;
+
+import java.math.BigInteger;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class Location {
+    String RegistryAddress;
+    String ResolverAddress;
+    Network NetworkId;
+    String Blockchain;
+    String Owner;
+    String BlockchainProviderURL;
+
+    private static final String addressFormat = "0x%040X";
+
+    public Location() {}
+
+    public void setRegistryAddress(String address) {
```

Comment:
Thanks for flagging it. I agree that this class should be cleaned up.

There are two things going on here:
1. I added some convenience setters, to set string addresses from `BigInt`. But these setters override the ones generated by Lombok. I guess it just doesn't generate these methods even though it should be able to. So I ended up adding two sets of setters.
2. `@AllArgsConstructor` generates a non-default constructor. So the default one without parameters is gone.

I've added a builder to avoid constructor issues. And I moved the convenience method to `Utilities`. So now it should look better.